src/Content/Text/HTML.php

```diff
@@ -688,4 +705,327 @@ public static function toMarkdown($html)
 
 		return $markdown;
 	}
+
+	/**
+	 * @brief Convert video HTML to BBCode tags
+	 *
+	 * @param string $s
+	 */
+	public static function htmlToBBVideo($s)
```

Comment:
The `html` prefix is redundant with the class name, please rename to `toBBCodeVideo` or `videoTagsToBBCode`.